sopt/src/main/scala/dagr/sopt/cmdline/ClpArgumentDefinitionPrinting.scala

```diff
@@ -98,7 +101,7 @@ object ClpArgumentDefinitionPrinting {
     val label = new StringBuilder()
     if (shortName.nonEmpty) label.append("-" + shortName + " " + shortType + ", ")
     label.append("--" + name + longType)
-    stringBuilder.append(wrapString(KGRN, label.toString(), KNRM))
+    stringBuilder.append(StringUtil.wrapString(KGRN, label.toString(), KNRM))
```

Comment:
Just thinking out loud - perhaps the terminal codes could become case classes that extend a trait like this:
```scala
trait TermCode {
    val code: String;
    def apply(String s): String = code + s + "\u001B[0m"
}

case class KGRN(override val code = ""\u001B[32m"")

// usage then looks like
builder.append(KGRN("my green text"))
```